BackofficeBundle/Resources/public/coffee/formBehavior/Token.coffee

```diff
@@ -0,0 +1,107 @@
+###*
+ * @namespace OpenOrchestra:FormBehavior
+###
+window.OpenOrchestra or= {}
+window.OpenOrchestra.FormBehavior or= {}
+
+###*
+ * @class Token
+###
+class OpenOrchestra.FormBehavior.Token extends OpenOrchestra.FormBehavior.AbstractFormBehavior
```

Comment:
``Token`` is not full of sense, can you rename it into something more oriented on what its doing ? (I don't get the context of what this behavior add to a form with that name)